.github/workflows/integration.yml

```diff
@@ -208,3 +208,161 @@ jobs:
           ADBC_POSTGRESQL_TEST_URI: "postgres://localhost:5432/postgres?user=postgres&password=password"
         run: |
           ./ci/scripts/python_test.sh "$(pwd)" "$(pwd)/build"
+
+  # ------------------------------------------------------------
+  # C/C++ driver service integration tests
+  # ------------------------------------------------------------
+  drivers-integration-conda:
+    name: "C/C++ (Conda) Driver Service Integration Tests"
+    runs-on: ubuntu-latest
+    services:
+      postgres:
```

Comment:
Remove all the postgres/dremio stuff? They're separate pipelines now

Reply:
good point, i'll do so